### ISSUE TITLE
Missing on_delete kwargs for ForeignKey property

### ISSUE THREAD

**Issue by evilpan** [phith0n/mooder#27]
archives/models.py: 127
```
parent = models.ForeignKey('self', verbose_name='父评论', blank=True, null=True)
```
It's worth noting that on_delete will be a required arg in Django 2.0 and omitting it causes warning for Django1.x.

**Comment by pannzh** [comment]
BTW, the dependencies in `requirements.txt` should better freeze the version number too, or it may cause unexpected issue for incompatible update.

**Comment by phith0n** [comment]
fixed on #37
